sdk/search/azure-search-documents/azure/search/documents/_generated/aio/operations/_documents_operations.py

```diff
@@ -257,9 +251,6 @@ async def search_post(
         :type search_request: ~azure.search.documents.models.SearchRequest
         :param request_options: Parameter group.
         :type request_options: ~azure.search.documents.models.RequestOptions
-        :keyword api_version: Api Version. The default value is "2021-04-30-Preview". Note that
-         overriding this default value may result in unsupported behavior.
-        :paramtype api_version: str
```

Comment:
@iscai-msft is this a known change in autorest or is this the result of some misconfiguration in the code gen?

Reply:
yes this is. We realized we had client parameters scattered in too many places, so we removed documentation for client params on methods